PalasoUIWindowsForms/ImageToolbox/ArtOfReadingChooser.cs

```diff
@@ -218,10 +227,98 @@ private void ArtOfReadingChooser_Load(object sender, EventArgs e)
 		   // _searchTermsBox.Text = @"flower";
 			_searchButton_Click(this,null);
 #endif
-
+			var searchControl = _searchTermsBox.TextBox.TheControl;
+			if (searchControl is TextBox)
+				searchControl.MinimumSize = _searchTermsBox.Size;
```

Comment:
It's not obvious why this isn't done in designer or why it might not be a text box. A comment along the lines, "On Linux the search control contains an HTML control rather than just a text box." would be a beginning.